cdpctl/validation/infra/issue_templates.yml

```diff
@@ -0,0 +1,137 @@
+summary: "The role ({0}) requires the following actions for the Datalake S3 bucket ({1})."
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_S3_RESOURCES
+summary: "The role ({0}) requires the following actions for all S3 resources"
+render_type: list
+---
+id: AWS_ROLE_FOR_DATA_BUCKET_MISSING_ACTIONS
+summary: "The role ({0}) requires the following actions for the S3 data location ({1})."
+render_type: list
+---
+id: AWS_IDBROKER_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
+summary: "The role ({0}) requires the following actions for resource wildcard (*)."
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_EC2_RESOURCES
+summary: "The role ({0}) requires the following actions for all EC2 resources ([*])"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
+summary: "The role ({0}) requires the following actions for all resources ([*])"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_SERVICE_ROLL_RESOURCES
+summary: "The role ({0}) requires the following actions for all resources (\"arn:aws:iam::*:role/aws-service-role/*\")"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_PATH
+summary: "The role ({0}) requires the following actions for the log storage path ({1}):"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_BUCKET
+summary: "The role ({0}) requires the following actions for the log storage bucket ({1}):"
+render_type: list
+---
+id: AWS_LOGGER_INSTANCE_PROFILE_SHOULD_CONTAIN_LOGGER_ROLE
+summary: "The logger instance profile {0} set in config {1} should contain a logger role."
+---
+id: AWS_LOGGER_ROLE_SHOULD_HAVE_EC2_TRUST
+summary: "The logger role {0} should contain a trust policy for EC2"
+---
+id: AWS_S3_BUCKET_INVALID
+summary: "The s3a url {0} is invalid."
+---
+id: AWS_S3_BUCKET_DOES_NOT_EXIST
+summary: "S3 bucket {0} does not exist."
+---
+id: AWS_S3_BUCKET_FORBIDDEN_ACCESS
+summary: "S3 bucket {0} has forbidden access."
+---
+id: AWS_NON_CCM_DEFAULT_SG_NEEDS_TO_ALLOW_CDP_CIDRS
+summary: "When not using CCM (tunnel = false), the default security group {0} is missing TCP port 9443 to be allowed access from the following Cloudera CDP CIDRs"
+render_type: list
+---
+id: AWS_NON_CCM_GATEWAY_SG_MISSING_CIDRS
+summary: "When not using CCM (tunnel = false), the gateway security group {0} is missing the following access:"
+render_type: list
+---
+id: AWS_DEFAULT_SG_NEEDS_ALLOW_ACCESS_INTERNAL_TO_VPC
+summary: "Your Default Security Group {0} should allow access to all TCP and UDP ports (0-65535) internal to the VPC"
+---
+id: AWS_GATEWAY_SG_NEEDS_ALLOW_ACCESS_INTERNAL_TO_VPC
+summary: "Your Gateway Security Group {0} should allow access to all TCP and UDP ports (0-65535) internal to the VPC"
+---
+id: AWS_SSH_KEY_ID_DOES_NOT_EXIST
+summary: "SSH key id ({0}) do not exist."
+---
+id: AWS_SSH_IS_INVALID
+summary: "SSH Key ID ({0}) is not valid."
+---
+id: AWS_NOT_ENOUGH_SUBNETS_PROVIDED
+summary: "Not enough {0} Subnets provided, at least 3 subnets required."
```

Comment:
I couldn't find where this is used. Is {0} going to be the number of subnets found?

Reply:
If it's just a placeholder for now, that's fine.

Reply:
No for that message its allowing us to pass in Public or Private to reuse

Reply:
Used in validate_aws_subnets.py line 91

Reply:
Nice!